docs/cookbook/custom-promotion-rule.rst

```diff
@@ -37,7 +36,7 @@ The new Rule needs a RuleChecker class:
          */
         public function getConfigurationFormType()
```

Comment:
`getConfigurationFormType ()` method was removed from the `RuleCheckerInterface`, so you should remove this method from the example.